settings.py

```diff
@@ -716,12 +716,6 @@ def get_locales():
         ),
         'output_filename': 'build/styles/fellowship.css',
     },
-    'mdn10': {
-        'source_filenames': (
-            'css/mdn10.css',
-        ),
-        'output_filename': 'build/styles/mdn10.css',
-    },
```

Comment:
Where was this bundle used? I don't notice any usage on master. Was the compressed file referenced directly somewhere?